secrets/standup-endpoints-config-secret.yaml

```
@@ -0,0 +1,21 @@
+# Format of <CONFIG> is:
+# {
+# "aws": {
+#    "context": "giantswarm-<CP-name>",
+#    "endpoint": <ENDPOINT>,
+#    "token": <TOKEN>,
+#   },
+# "other-provider": {
+#    "context": "giantswarm-<CP-name>",
+#    "endpoint": <ENDPOINT>,
+#    "token": <TOKEN>,
+#   },
+# }
```

Should be YAML I think:
```suggestion
aws:
  context: giantswarm-<CP-name>
  endpoint: <API endpoint URL with protocol prefix>
  token: <API token (uuid)>
other-provider:
  context: giantswarm-<CP-name>
  endpoint: <API endpoint URL with protocol prefix>
  token: <API token (uuid)>
```

---

Good point. Funnily, it worked with json too 😄